docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -309,25 +309,30 @@ Upon diagnosis with multiple myeloma, simulants should be assigned values for ea
   * - Male
     - Under 65
     - 0.211
-    - 0.872
-    - 0.081
+    - uniform distribution from 0.17 to 0.34
+    - 0.25
   * - Male
     - 65+
     - 0.159
-    - 0.872
-    - 0.081
+    - uniform distribution from 0.17 to 0.34
+    - 0.25
   * - Female
     - Under 65
     - 0.225
-    - 0.872
-    - 0.081
+    - uniform distribution from 0.17 to 0.34
+    - 0.25
   * - Female
     - 65+
     - 0.165
-    - 0.872
-    - 0.081
+    - uniform distribution from 0.17 to 0.34
+    - 0.25
+
+The probability of high cytogenetic risk (uniform distribution between 17% and 30%) was obtained from [Rice-et-al-2020]_. The lower bound is the proportion of patients tested who had high cytogenetic risk, and the upper bound is the proportion if those who were tested and had undocumented/unknown cytogenetic risk and were reallocated assuming the same proportion of high:standard risk among those with documented/known cytogenetic risk.
+
+
+The probability of renal impairment (25%) was obtained from [Mikhael-et-al-2021]_. Notably, this paper defined renal impairment as eGFR <=50 ml/min/m^2, which will be an underestimate of the proportion of patients with renal impairment defined as eGFR <=50 ml/min/m^2, which is the definition of renal impairment used to inform our risk effect.
```

Comment:
@alibow I heard from Megan that the prevalence of renal impairment was estimated to be around 40% for eGFR <= 60 based on Flatiron data. Are we waiting for her follow-up on this?

Reply:
Yes, we are!

Reply:
Updated :)